### PR TITLE
[plugin.video.noco] v1.0.13

### PR DESCRIPTION
**Pull request by pcjco**
### Description
  - Navigation in list of videos now shows N items per page and navigations items (Next page, Previous page)
  - Add "Free content" entry to see free content of other catalogs (when logged as user) 
  - Add setting to use in guest mode to see free content without being logged (some recent videos from Nolife catalog)
  - Adding "History" entry to view previously seen video
  - "Most popular" entry allows to choose the period of popularity (week, month, all time) 
  - Adding "Top Rated" entry 
  - Renaming "Playlists" entry by "Noco playlists" (only "File d'attente" inside atm)
  - Adding basic support on "Noco playlists" : adding/deleting/moving videos & clear/queue playlist
  - Adding "Noco favorite shows" entry with basic support (adding/deleting favorite shows, clear favorites)
  - Already seen videos are now filtered directly within noco api request
  - Random video is now asked using a noco api request (faster)
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.noco/addon.xml [hunk 1]
```diff
@@ -18,8 +18,8 @@
     <email>julien@gormotte.info</email>
     <summary lang="fr">Plugin pour noco.tv</summary>
     <summary lang="en">Plugin for noco.tv</summary>
-    <description lang="fr">Un plugin pour acceder aux services de noco.tv. Necessite un compte pour fonctionner</description>
-    <description lang="en">A plugin to access noco.tv services. An account is needed for the plugin to work</description>
+    <description lang="fr">Un plugin pour accéder aux services de noco.tv. Nécessite un compte pour fonctionner. Un mode invité est disponible avec un contenu limité.</description>
+    <description lang="en">A plugin to access noco.tv services. An account is needed for the plugin to work. A guest mode is available with limited content.</description>
     <disclaimer lang="fr">Ce plugin n'est ni publié ni maintenu par noco, et donc non officiel.</disclaimer>
     <disclaimer lang="en">This plugin is not official. Any questions are for me and not noco stafff.</disclaimer>
```
Comment (eracknaphobia) [inline plugin.video.noco/addon.xml]: Here too

**Comment by pcjco** [comment]
Changes are done.

File: plugin.video.noco/addon.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.noco" name="noco" version="1.0.13" provider-name="Julien Gormotte (gormux)">
   <requires>
-    <import addon="xbmc.python" version="2.1.0"/>
+    <import addon="xbmc.python" version="2.1.14"/>
```
Comment (eracknaphobia) [inline plugin.video.noco/addon.xml]: The xmbc.python version should be 2.14.0

**Comment by eracknaphobia** [comment]
Still need the xbmc.python version to be 2.14.0

Everything else looks good.

**Comment by pcjco** [comment]
xbmc.python version set to 2.14.0 this time.

**Comment by eracknaphobia** [comment]
Thanks!

**Pull request merged**